src/core/sys/posix/pwd.d

```diff
@@ -196,10 +196,14 @@ else version( OpenBSD )
     int getpwnam_r(in char*, passwd*, char*, size_t, passwd**);
     int getpwuid_r(uid_t, passwd*, char*, size_t, passwd**);
 }
-else version (Solaris)
+else version( Solaris )
```

Comment:
Using this coding style is discouraged. I should probably get round to mechanically changing it to the right style.